docs_manual/source/metadata_advanced.rst

```diff
@@ -216,3 +218,40 @@ Protein Expression Profiling / Mass Cytometry Plugin
     * *Raw Data Files* and *Derived Data Files* are linked to ``{Assay Name}``
 - Used with measurement type / technology type
     * protein expression profiling / mass cytometry
+
+Universal Assay Plugin
+----------------------
+
+This plugin enables the user to define row-specific and inline links to
+iRODS collections via comments in the ``STUDY ASSAYS`` section of the ISA-Tab
+investigation file.
+
+- Internal name: ``samplesheets_assay_generic``
+- Row-specific links
+    * Place one or multiple comments starting with ``SODAR Assay Row Path``.
```

Comment:
We should make it clear here that the ordering of the comments matters. The user should be explicitly encouraged to number them in order.

In the code, you correctly do sorting of the dictionary keys to ensure this. However, despite the nice example below, this documentation could lead the user to think each comment name can be whatever, as long as it starts with the given string.